mockgen/tests/no_import_for_structs_in_interface/README.md

```diff
@@ -0,0 +1,35 @@
+Types that appear in the signature of a mocked interface are not imported
```

Comment:
This test case is phrased as a bug report. It will start to make less sense when someone reads it in the future. Rather than word it as "x is not working" you could state it as "verify that x works correctly".